src/actionCreators/resources.js

```diff
@@ -26,7 +26,7 @@ import _ from 'lodash'
 export const update = (resourceKey, currentUser, errorKey) => (dispatch, getState) => {
   const state = getState()
   const uri = findResourceURI(state, resourceKey)
-  const rdf = new GraphBuilder(state.selectorReducer.entities.resources[resourceKey], state.selectorReducer.entities.resourceTemplates).toTurtle()
+  const rdf = new GraphBuilder(state.selectorReducer.entities.resources[resourceKey], state.selectorReducer.entities.resourceTemplates).graph.toCanonical()
```

Comment:
Maybe you want two variables here: one to hold the turtle (for persisting) on line 30 and one to hold the n-triples (for checksumming) on line 31?

Reply:
Thanks, good catch! Just pushed a commit that use the `GraphBuilder` instance to produce both Turtle and n-triples serializations as needed.